plug-ins/dns_spoof/dns_spoof.c

```diff
@@ -19,6 +19,11 @@
 
 */
 
+/*
+ * May 2015
+ * Frekky - Added TTL option for spoofed DNS replies.
+*/
+
```

Comment:
Don't you think your credits would be better placed within the AUHTORS file rather than in the source code?